src/components/Camera/hooks/useFocusTap.ts

```diff
@@ -20,7 +29,8 @@ const useFocusTap = ( cameraRef: Object, supportsFocus: boolean ): Node => {
     } );
   }, [tappedCoordinates, focusOpacity] );
 
-  const onFocus = useCallback( async ( { x, y }: GestureResponderEvent ) => {
+  type TapEvent = GestureStateChangeEvent<TapGestureHandlerEventPayload>;
```

Comment:
Gesture-handler also have their own types. I am not sure if this is necessary but in any case `GestureResponderEvent` is not exported from gesture-handler.